sql/updates/world/master/9999_99_99_99_world.sql

```diff
@@ -34,7 +34,7 @@ INSERT INTO `conditions` (`SourceTypeOrReferenceId`, `SourceGroup`, `SourceEntry
 
 -- Isiset Text
 DELETE FROM `creature_text` WHERE `entry` IN (39587);
```

Comment:
Oh. I forgot to specify that you need to change this one too,
`entry` `->` `CreatureID`